src/main/java/org/broadinstitute/dsde/consent/ontology/cloudstore/StorageFactory.java

```diff
@@ -4,10 +4,9 @@
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.json.JsonFactory;
-import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.json.gson.GsonFactory;
```

Comment:
According to the package changelog, even Google is swapping out Jackson with Gson.

Reply:
Yes, after I made the change, I followed the commits in google and found that out too 👍🏽